packages/pn-pa-webapp/src/components/NewNotification/Attachments.tsx

```diff
@@ -49,14 +56,18 @@ const AttachmentBox: React.FC<AttachmentBoxProps> = ({
   fieldLabel,
   fieldValue,
   fieldTouched,
-  fieldErros,
+  fieldErrors,
   onFieldTouched,
   onFileUploaded,
   onRemoveFile,
   fileUploaded,
 }) => {
+  const [focused, setFocused] = useState(false);
   const { t } = useTranslation(['notifiche']);
+  const { t: tc } = useTranslation(['common']);
```

Comment:
Avoid to define another t function. Inlcude common in the array above and use the ns prop